docs/runbooks/troubleshooting-guide.md

```diff
@@ -0,0 +1,63 @@
+# Troubleshooting Guide
+
+## General information to always gather
+
+This is a basic set of information that is almost always relevant to troubleshooting.
+
+* Determine the version of Postgres, TimescaleDB, the Promscale connector, and the Promscale extension.
+  * Postgres: `select version();`
+  * Timescaledb and the Promscale extensions: `select extname, extversion from pg_extension where extname in ('timescaledb', 'promscale');`
+* Determine the chunk policy: `select _prom_catalog.get_default_chunk_interval();`
```

Comment:
Can we also mention how to get chunk and it's index size to configure chunk interval appropriately?